docs/source/best-practices/homequota.rst

```diff
@@ -0,0 +1,63 @@
+*******************************************************
+Stay Under your Quota at /home/
+*******************************************************
+It is vital to stay under your ``/home/<username>`` quota to prevent issues using Open OnDemand and performing tasks on Discovery. Here are some suggestions to stay under quota and general best practices with using your ``/home/<username>`` directory.
+
+Utlilize /scratch and /work Directories
+=======================================================
+For long-term research work storage, ``/work`` should be used. If your PI does not have space in ``/work`` setup, they can request it using the following form https://bit.ly/NURC-NewStorage and can request additional storage using https://bit.ly/NURC-StorageExtension. More details of ``/work`` can be found at :ref:`discovery_storage`. For temporary job files you can utilize /scratch/<username>. Please be mindful of the purge policy of ``/scratch`` which is covered in :ref:`discovery_storage.rst`.
+
+Determine Large Files/Directories in /home/<username>
+=======================================================
+From a compute node, ``srun --pty /bin/bash``, run the following command from your ``/home/<username>`` directory ::
+    
+ du -shc .[^.]* *
```

Comment:
sample output?

Reply:
@visionjo I think we could add a sample output here but we might have to pick a $HOME directory that has some common files but not too filled because I don't think it would be helpful to have a large output here. I could be wrong on that though.

Reply:
@visionjo and @manasvita , I have added a code block (same style from Joe's unix updates) and added a sample output from a directory.